features/see_dishes.feature

```diff
@@ -0,0 +1,12 @@
+Feature: As a visitor
+  In order to choose dishes
+  I need to be able to see a menu with dishes
+
+  Scenario: Visitor see dishes
+    When I visit the dishes page
+    # Then I should see the page
+    Then I see "Main Courses"
+    And I see "Meatballs"
+    And I see the price of "$8.99"
+    And I see "Add to" button
```

Comment:
Why is this button "Add to"? Shouldn't it be something like "Add to Cart" or "Add to Order"?